README.md

```diff
@@ -318,3 +318,30 @@ m.it("fails", sub()
     m.fail()
 end sub)
 ```
+
+#### `m.addContext(ctx as object)`
+Provides context to cases in the form of extra fields.
+
+- `ctx` should be a `roAssociativeArray`,
+- `m.addContext` can be called multiple times,
+- Context cascades into sub-suites.
+
+Example:
+```brightscript
+m.addContext({
+    aField: "value 1"
+})
+
+m.describe("a test suite with context", sub()
+    m.addContext({
+        afn: function()
+            return "value 2"
+        end function
+    })
+
+    m.it("a test case", sub()
+        m.assert.equal("value 1", m.aField, "Get context field")
+        m.assert.equal("value 2", m.aFunction(), "Use context function")
```

Comment:
I assume this should be `m.afn()`, so I'll fix that before I merge 😄 

```suggestion
        m.assert.equal("value 2", m.afn(), "Use context function")
```

Reply:
🤦